CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - `elasticsearchreceiver`: Use same metrics as JMX receiver for JVM metrics (#7160)
 - `elasticsearchreceiver`: Implement scraping logic (#7174)
 - `datadogexporter`: Add http.status_code tag to trace stats (#6889)
+- `datadogexporter`: Add configuration option to use OTel span name into the Datatog resource name (#6611)
```

Comment:
Should we remove it from the `v0.42.0` changelog then? (Line 76)

Reply:
Thank you for pointing me that! I removed it.